doc/api/crypto.md

```diff
@@ -972,6 +1130,9 @@ The `key` is the raw key used by the `algorithm` and `iv` is an
 [buffers][`Buffer`].
 
 ### crypto.createDiffieHellman(prime[, prime_encoding][, generator][, generator_encoding])
```

Comment:
I think 0.11.12 does make more sense. If someone wants to look at the 0.10 docs to find the original that's fine.